rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -422,7 +406,7 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
             if(query && query.sortBy && xfilters[query.sortBy]){
                 order(xfilters[query.sortBy],query.sortOrder=='ascending'?'asc':'desc')
             }else{
-                if(paginationEnabled) {
+                if(query.paginatedRequired) {
```

Comment:
I think the groupPath ordering doesn't have to be only if pagination is required.  Suggest we just include the `order("groupPath", "asc")` for all queries

Reply:
Doesn't this change the previous expected behavior? May be for the next version...